.github/workflows/ci.yaml

```diff
@@ -113,6 +117,7 @@ jobs:
         env:
           REPORT: ${{ steps.slither.outputs.stdout }}
         with:
+          github-token: ${{ secrets.GH_TOKEN }}
```

Comment:
I don't think we need ot use GH_TOKEN, instead we should set
`permissions:
   pull-request: write`

Reply:
permissions are already set

Reply:
Shouldn't in that case we be using secrets.GITHUB_TOKEN? We don't have GH_TOKEN defined